File: src/main/java/client/database/data/PlayerDiseaseData.java
package client.database.data;

public class PlayerDiseaseData {
   private int disease;

   private int mobSkillId;

   private int mobSkillLevel;

   private int length;

   public PlayerDiseaseData(int disease, int mobSkillId, int mobSkillLevel, int length) {
      this.disease = disease;
      this.mobSkillId = mobSkillId;
      this.mobSkillLevel = mobSkillLevel;
      this.length = length;
   }

   public int getDisease() {
      return disease;
   }

   public int getMobSkillId() {
      return mobSkillId;
   }

   public int getMobSkillLevel() {
      return mobSkillLevel;
   }

   public int getLength() {
      return length;
   }
}

File: engine/model/src/main/scala/client/database/data/PlayerDiseaseData.scala
package client.database.data

class PlayerDiseaseData(private var _disease: Int, private var _mobSkillId: Int, private var _mobSkillLevel: Int,
                        private var _length: Int) {
  def disease: Int = _disease

  def mobSkillId: Int = _mobSkillId

  def mobSkillLevel: Int = _mobSkillLevel

  def length: Int = _length
}
